client/verta/docs/_static/custom.css

```
@@ -71,9 +71,9 @@ html {
 body {
   display: flex;
   flex-direction: column;
-  height: 100%;
   margin: 0;
   background: #f5f7f9;
+  min-height: 100%;
```

This solves the problem for me in Firefox & Safari,
though I'm not sure what the best practice is regarding `vh` vs `%`.
```suggestion
  min-height: 100vh;
```